README.md

```diff
@@ -120,10 +123,36 @@ Debugr can handle structured data in addition to plain string messages, but care
 when logging such structured data, because some handlers will not process the data immediately;
 if the application keeps a reference to the logged data and then mutates it in any way, there's
 a chance that what ends up being logged is the mutated data, as opposed to the data as it was
-at the time it was passed to the logger. An easy remedy is to never log data which is mutated
-farther down the road, but if you do need to log something and then mutate it later, you can
-take a snapshot of the data when logging it. There's a helper function called `clone()` exported
-from `@debugr/core` which you can use for this purpose if needed.
+at the time it was passed to the logger. There are three ways you can deal with this problem:
```

Comment:
opravdu obsahle, ale why not. Najs

Reply:
jasný no.. čtu to teď znova a vlastně fakt nevím co by šlo vynechat..

Reply:
proc neco vynechavat? :) Nech to jak to je ;)

Reply:
no protoze je to dlouhy :-D ale ok ;-)